client/src/configs/env.ts

```diff
@@ -53,3 +53,7 @@ export const EXPRESS_OAUTH_LOGOUT_URL =
 
 /** Do you want to disable login protection? */
 export const DISABLE_LOGIN_PROTECTION = process.env.REACT_APP_DISABLE_LOGIN_PROTECTION === 'true';
+
+/** Activate the product-catalogue menu */
+export const ENABLE_PRODUCT_CATALOGUE = process.env.REACT_APP_ENABLE_PRODUCT_CATALOGUE === 'true';
+export type ENABLE_PRODUCT_CATALOGUE = typeof ENABLE_PRODUCT_CATALOGUE;
```

Comment:
Are we using this exported type? Other types have not been declared/exported

Reply:
seems this is now a redundant syntax for exporting the varaible's, type. [old habits](https://github.com/onaio/reveal-frontend/blob/master/src/configs/env.ts).